kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/ApplyMojo.java

```diff
@@ -206,7 +206,7 @@ public void executeInternal() throws MojoExecutionException {
 
             boolean openShift = OpenshiftHelper.isOpenShift(kubernetes);
             if (openShift) {
-                getLog().info("OpenShift platform detected");
+                log.info("[[B]]OpenShift[[B]] platform detected");
```

Comment:
`getLog()` is retrieving the log defined in AbstractMojo which is **not** an instance of KitLogger.

This change is OK for now, but once we tackle #147 and #146 [these variables](https://github.com/eclipse/jkube/blob/edb82c8ef8dff52a48b4c4e5dac4b183d5fe61e1/kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractJKubeMojo.java#L70) should be private and getter methods (`getKitLogger()`, etc.) should be used instead